README.md

```diff
@@ -191,6 +191,22 @@ For users with NuFi extension installed, there are no specific actions required.
 Simply access `window.cardano.nufi` from anywhere as it is not controlled by
 the NuFi Widget SDK.
 
+## Troubleshooting
+
+### Black and non-responsive Widget
+We noticed that accessing `window.cardano.nufiSSO` directly after calling
+`initNufiDappCardanoSdk` can lead to "frozen/black" Widget window.
+If that is the case for you, please call any asynchronous operation
+after calling `initNufiDappCardanoSdk` and before accessing `window.cardano.nufiSSO`.
+Something like this should fix the problem:
+
+```typescript
+initNufiDappCardanoSdk(nufiCoreSdk, 'sso', {provider: 'google'})
+await new Promise((resolve) => setTimeout(resolve, 100))
+const api = await window.cardano.nufiSSO.enable()
+```
+This problem should be fixed soon.
```

Comment:
for visibility, I'd consider making an issue in the repo which could be referenced here so that whoever reads this has visibility into the issue status/has a chance to ask or provide further insight, e.g. if they experience the same issue